src/stubs.ts

```diff
@@ -309,12 +356,16 @@ export const createFigma = (config: TConfig): PluginAPI => {
     dashPattern: ReadonlyArray<number>;
     fillStyleId: string | PluginAPI["mixed"];
     strokeStyleId: string;
+    strokeGeometry: VectorPaths;
+    fillGeometry: VectorPaths;
     outlineStroke() {
       return null;
     }
   }
   class RectangleNodeStub {
     type = "RECTANGLE";
+    pluginData: { [key: string]: string } = {};
```

Comment:
I ended up adding a bunch of `pluginData` and `sharedPluginData` initializations because `applyMixins` wasn't adding default property values and it was causing `Object.create` to fail. A better solution would probably be to handle that in applyMixins, but I wasn't able to fully-grasp what that function was doing 😅